docs/configuration/initialconfig/automatic.md

```diff
@@ -0,0 +1,85 @@
+# Automatic Configuration
+
+Creation of the dedicated Moodle user and role, as well as the setup of the
+webservice for the archive worker, can be done automatically.
+
+The easiest way is to use the automatic configuration feature provided via the
+Moodle admin interface but a fully automated configuration via CLI is also
+supported.
+
+
+## Using the Moodle Admin Interface
+
+!!! info
```

Comment:
I am confused: Why would you recommend this for "most users"?
As you are writing in the paragraph above, it is the easiest way to use the automatic configuration?